PF/Assignment1/assignment6.java

```diff
@@ -0,0 +1,62 @@
+package programing_fundamental;
+import java.util.*;
+
+public class assignment6 {
+	public int[] join(int a[], int asize, int b[], int bsize, int c[])
+	{
+		int i,j,l;
+		int k=0;
+		for(i=0,j=0;i<asize&&j<bsize;i++,j++)
+		{
+			if(a[i]<b[j])
+			{
+				c[k++]=a[i];j--;
+			}
+			else if(b[j]<a[i])
+			{
+				c[k++]=b[j];i--;
+			}
+			else
+			{
+				c[k++]=a[i];
+				c[k++]=b[j];
+			}
+		}
+		if(i==asize&&j!=bsize)
```

Comment:
ACTION: No need of another for loop.This could be done in the above loop itself. Use "||" in the condition part instead of "&&". So that it will be run until any of the array has elements remaining.